### PR TITLE
[CLOUDCRAFT-766] Component: Network ACL

### PR DESCRIPTION
**Pull request by jamesponddotco**
### What does this PR do? What is the motivation?
Adds documentation for a new component in Cloudcraft, the Network ACL component. The documentation follows the same style we've been using for other components.

### Merge instructions
Please wait until the Cloudcraft team has reviewed the content before going ahead with the review and merging of the PR.

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by kayayarai** [comment]
Editorial review tracked in https://datadoghq.atlassian.net/browse/DOCS-8381

**Review by srividya-rajagopalan — APPROVED** [review]

File: content/en/cloudcraft/components-aws/network-acl.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+Use the Network ACL component to visualize network access control lists from your Amazon Web Services architecture.
+
+{{< img src="cloudcraft/components-aws/network-acl/component-network-acl-diagram.png" alt="Screenshot of an isometric Cloudcraft diagram showing the 'Network ACL' AWS component." responsive="true" style="width:60%;">}}
+
+## Toolbar
+
+Use the toolbar to configure and customize the component. The following options are available:
+
+- **Visibility**. Toggle the visibility of the component on the diagram.
```
Comment (aliciascott) [inline content/en/cloudcraft/components-aws/network-acl.md:15]:
```suggestion
- **Visibility**: Toggle the visibility of the component on the diagram.
```

File: content/en/cloudcraft/components-aws/network-acl.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+- **Color**. Select a fill color for the body of the component and an accent color for its symbol. You can use the same colors on 2D and 3D views or different colors for each.
```
Comment (aliciascott) [inline content/en/cloudcraft/components-aws/network-acl.md:16]:
```suggestion
- **Color**: Select a fill color for the body of the component and an accent color for its symbol. You can use the same colors on 2D and 3D views or different colors for each.
```

File: content/en/cloudcraft/components-aws/network-acl.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- **Name**. Give the NACL a name.
```
Comment (aliciascott) [inline content/en/cloudcraft/components-aws/network-acl.md:17]:
```suggestion
- **Name**: Give the NACL a name.
```

File: content/en/cloudcraft/components-aws/network-acl.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+- **Shape**. Select a shape for the component.
```
Comment (aliciascott) [inline content/en/cloudcraft/components-aws/network-acl.md:18]:
```suggestion
- **Shape**: Select a shape for the component.
```

File: content/en/cloudcraft/components-aws/network-acl.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- **Padding**. Increase or decrease the amount of space inside the component.
```
Comment (aliciascott) [inline content/en/cloudcraft/components-aws/network-acl.md:19]:
```suggestion
- **Padding**: Increase or decrease the amount of space inside the component.
```

File: content/en/cloudcraft/components-aws/network-acl.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+- **Rules**. Add, edit, or remove inbound and outbound rules for the NACL.
```
Comment (aliciascott) [inline content/en/cloudcraft/components-aws/network-acl.md:20]:
```suggestion
- **Rules**: Add, edit, or remove inbound and outbound rules for the NACL.
```

File: content/en/cloudcraft/components-aws/network-acl.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+        "2d": "#5698ff"
+    },
+    "link": "https://aws.amazon.com/",
+    "locked": true
+}
+```
+
+- **type: string**: The type of component.
+- **id: string**: A unique identifier for the component consisting of the string `acl-` followed by a random 17-character alphanumeric string.
+- **arn: string**: The globally unique identifier for the component within AWS, known as [Amazon Resource Names](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html).
```
Comment (aliciascott) [inline content/en/cloudcraft/components-aws/network-acl.md:91]:
```suggestion
- **arn: string**: The globally unique identifier for the component within AWS, known as [Amazon Resource Names][2].
```

File: content/en/cloudcraft/components-aws/network-acl.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+  - **target: string**: The destination for the traffic (CIDR range).
+  - **targetType: string**: The target type for the rule. Accepts either `ip` or `ipv6`.
+  - **access: string**: The type of access for the rule. Accepts either `allow` or `deny`.
+- **color: object**: The fill color for the component body.
+  - **isometric: string**: The fill color for the component in the 3D view. Must be a hexadecimal color. Defaults to `#ECECED`.
+  - **2d: string**: The fill color for the component in the 2D view. Must be a hexadecimal color. Defaults to `#CC2264`.
+- **link: uri**: Link the component to another diagram using the `blueprint://ID` format or to an external website using the `https://LINK` format.
+- **locked: boolean**: If `true`, changes made to the component using the application are disabled until unlocked.
+
+[1]: /cloudcraft/api/
```
Comment (aliciascott) [inline content/en/cloudcraft/components-aws/network-acl.md:118]:
```suggestion
[1]: /cloudcraft/api/
[2]: https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @jamesponddotco looks good just a few minor suggestions!

File: content/en/cloudcraft/components-aws/network-acl.md (hunk 6 above)
Comment (ruihanzhao) [reply content/en/cloudcraft/components-aws/network-acl.md:20]: The rules here are read-only. So actually I would change this to "View inbound and outbound rules for the NACL"

**Comment by jamesponddotco** [comment]
@aliciascott The suggestions have been applied; sorry for the delay :S

**Review by aliciascott — APPROVED** [review]
looks good! @jamesponddotco Let me know if this is ready to be merged?

**Comment by jamesponddotco** [comment]
@aliciascott Ready when you're :)

**Pull request merged**